09-MasterDetail/src/PrismSample/Views/HomePage.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<MasterDetailPage xmlns="http://xamarin.com/schemas/2014/forms"
+                  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                  xmlns:prism="http://prismlibrary.com"
+                  x:Class="PrismSample.Views.HomePage" 
+                  MasterBehavior="Popover"
+                  IconImageSource="hamburger.png">
```

Comment:
this is in the wrong place as it should be on the Master not the MasterDetailPage